crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -235,8 +235,7 @@ impl<'w, 's> Commands<'w, 's> {
         }
     }
 
-    /// Pushes a [`Command`] to the queue for creating a new empty [`Entity`],
-    /// and returns its corresponding [`EntityCommands`].
+    /// Reserves a new empty [`Entity`] to be spawned, and returns its corresponding [`EntityCommands`].
```

Comment:
`its corresponding` doesn't quite right, maybe `an associated` or something?

Reply:
The other spawn methods word this as `its corresponding` as well. Another option:
```suggestion
    /// Reserves a new empty [`Entity`] to be spawned, and returns it in an [`EntityCommands`].
```

Reply:
I like the wording in this PR better :)